lib/bootloader_setup.pm

```diff
@@ -448,6 +448,16 @@ sub bootmenu_network_source {
                 # As we use boot options, no extra action is required
                 return;
             }
+
+            # fate#322276
+            # Point linuxrc to a repomd repository and load the installation system from local medium
+            if (($m_protocol eq "http") && (get_var('REMOTE_REPOINST'))) {
+                type_string_very_slow " install=$m_mirror";
+                # Specifies the installation system to use, e.g. from where to load installer
+                type_string_very_slow ' instsys=disk:/boot/x86_64/root';
```

Comment:
What's about using ARCH variable in the path so we can run this on qemu backend on other architectures?
Otherwise, let's add some `die` call when trying to use for different architecture/backend.

Reply:
I'm up for the architecture independant (adding `ARCH`)

Reply:
ah, good catch!

Reply:
thanks guys for pointing it out!